### DISCUSSION TITLE
[v4] `addComponents`=`addUtilities`? `@layer components` is not being used.

### DISCUSSION THREAD

**Discussion by saadeghi** [tailwindlabs/tailwindcss#14795] [General]
It seems like the output of `addComponents` and `addUtilities` is the same in CSS.
There's a `components` layer in output CSS:
```css
@layer theme, base, components, utilities;
```
but looks like there's no way to add styles to it.  
`addComponents` adds styles to `utilities` layer which gives the components an equal specificity as utilities.

Shouldn't `addComponents` add styles to `@layer components`?

**Comment by stafyniaksacha** [comment]
I think it will be the case when looking at migration script tests here: https://github.com/tailwindlabs/tailwindcss/blob/ca4e4aebd8cc896ccd3f55f43b3fa68af9ad1baf/packages/%40tailwindcss-upgrade/src/codemods/migrate-missing-layers.test.ts#L169-L174
